libraries/AP_Baro/AP_Baro_MS5611.h

```diff
@@ -50,7 +51,7 @@ class AP_Baro_MS56XX : public AP_Baro_Backend
         return probe(baro, std::move(dev), BARO_MS5637);
     }
     static AP_Baro_Backend *probe_5837(AP_Baro &baro, AP_HAL::OwnPtr<AP_HAL::Device> dev) {
-        return probe(baro, std::move(dev), BARO_MS5837);
+        return probe(baro, std::move(dev), BARO_MS5837_30BA);
```

Comment:
```suggestion
        return probe(baro, std::move(dev), BARO_MS5837);
```
The functionality is less confusing if the probe is shown as being done on the general type, and then a variant being decided on later.